src/stash-list/window.vue

```diff
@@ -104,6 +105,10 @@ export default defineComponent({
             return "Unstashed Tabs";
         },
 
+        tooltip(): string {
+            return `${this.title} (${this.validChildren.length})`;
+        },
```

Comment:
I think you can just say `N unstashed tabs` (the tooltip doesn't have to directly match the title, since the title never changes).

Reply:
The title differs based on whether it's a list of all tabs or unstashed tabs though. So:

```
`${this.validChildren.length} ${this.title}`
```

=> `10 Open Tabs`

=> `10 Unstashed Tabs`

Is that what you'd want?

Reply:
Ah good point. Yeah I think that works fine.